pkg/upstream/http.go

```diff
@@ -92,6 +95,19 @@ func (h *httpUpstreamProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 	}
 }
 
+type UnixRoundTripper struct {
+	Transport *http.Transport
+}
+
+func (t *UnixRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
```

Comment:
Please add comments to explain anything that is exported

Reply:
My bad, it shouldn't be exported, I'll change that.

Reply:
Not exported anymore :)